web/src/settings_components.ts

```diff
@@ -146,13 +229,19 @@ export function change_element_block_display_property(elem_id, show_element) {
     }
 }
 
-export function is_video_chat_provider_jitsi_meet() {
-    const video_chat_provider_id = Number.parseInt($("#id_realm_video_chat_provider").val(), 10);
+export function is_video_chat_provider_jitsi_meet(): boolean {
+    const video_chat_provider_id = Number.parseInt(
+        $<HTMLSelectElement & {type: "select-one"}>("select#id_realm_video_chat_provider").val()!,
```

Comment:
For now, I have added this type casts only where it is needed. Do we want to include this in all the jquery objects?

Reply:
Only where needed seems correct.